src/server/lang.js

```diff
@@ -0,0 +1,43 @@
+import path from "path"
+import {readdir} from "fs"
+
+import {addLocaleData} from "react-intl"
+import fr from "react-intl/locale-data/fr"
+import en from "react-intl/locale-data/en"
+
+const DEFAULT_LANGUAGE = 'fr'
+
+export function loadTranslations(langDir){
+  addLocaleData(...fr)
+  addLocaleData(...en)
+
+  var messages = new Map()
+  var dotIndex = -1
+  var json
+  readdir(langDir, (err, files) => { 
+    if(err){
+      console.log("Unable to load translation files.", err)
```

Comment:
1. This is not a `log` but an `error`.
2. Can you actually continue? Are you sure you shouldn't `throw`?
3. Improve the error message by explaining which `langDir` you were trying to load.